### PR TITLE
feat: adding suspicious and pooled asset icons to cards + rarity

### PR DESCRIPTION
**Pull request by JackShort**
* added the appropriate icons for suspicious and pooled asset
* added rarity

### CODE REVIEW

File: src/nft/components/collection/CollectionAsset.tsx [hunk 1]
```diff
@@ -68,13 +75,23 @@ export const CollectionAsset = ({
           <Card.PrimaryRow>
             <Card.PrimaryDetails>
               <Card.PrimaryInfo>{asset.name ? asset.name : `#${asset.tokenId}`}</Card.PrimaryInfo>
+              {asset.openseaSusFlag && <Card.Suspicious />}
             </Card.PrimaryDetails>
+            {asset.rarity && provider && provider.rank && (
+              <Card.Ranking
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionAsset.tsx:81]: Not seeing ranking on any cards atm

File: src/nft/components/collection/Card.tsx [hunk 2]
```diff
@@ -106,7 +117,6 @@ const Image = ({ uniformHeight, setUniformHeight }: ImageProps) => {
     <Box display="flex" overflow="hidden">
       <Box
         as={'img'}
-        alt={asset.name || asset.tokenId}
```
[the remark this answers is not in the document]
Comment (JackShort) [reply src/nft/components/collection/Card.tsx:109]: when it was loading the alt text would show up

File: src/nft/components/collection/CollectionAsset.tsx (hunk 1 above)
Comment (cbachmeier) [reply src/nft/components/collection/CollectionAsset.tsx:81]: Pulled latest and now seeing them

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**